### DISCUSSION TITLE
I like my laptop setup.  About to install to my desktop.  Have a couple questions about Hyprland, end-4, WMs and things i need to know.  Thanks for you help.

### DISCUSSION THREAD

**Discussion by bpawnzZ** [end-4/dots-hyprland#594] [Q&A]
"Hey, I'm not a noob to Linux or Arch Linux, but I'm still getting the hang of window managers, specifically Hyprland. I'm wondering about the best way to manage dotfiles like this. I know to back up .local and .home, but here's how I set up my system: I created a Python virtual environment for Python 3.10.8 with a new user, ran the entire setup within that environment, and sourced it in my .zshrc file. I chose this approach because I had issues with Conda not being able to install certain pip modules.

So, is this the right way to do things? Are there any best practices I should be following? Coming from KDE, I'm still figuring out the WM way of life. How do you guys set up your systems?"

Let me know if this looks good to you, or if you need any other suggestions

**Comment by bpawnzZ** [comment]
![photo_2024-06-08_09-24-41](https://github.com/end-4/dots-hyprland/assets/151177455/e3922849-ad7b-4a64-851b-c8332b7fa14d)
![photo_2024-06-08_09-24-38](https://github.com/end-4/dots-hyprland/assets/151177455/c542f160-b65d-4be1-a099-0f7704267476)
![photo_2024-06-08_09-24-49](https://github.com/end-4/dots-hyprland/assets/151177455/584a394f-5c8f-4dea-ae3c-50f849aef6c6)
![photo_2024-06-08_09-24-47](https://github.com/end-4/dots-hyprland/assets/151177455/f9e08fd3-a574-4cdd-92ac-6cbd99aab784)

Pretty dope on the laptop tho.

**Comment by Soliprem** [comment]
I think pipx would be an easier way to run the virtual env than creating a new user manually, but the best option is always pacman if the packages you need are there (and for this project, as far as I recall, they are). This approach has the potential to be a tad harder to maintain.. but hey, if it works, it works!
